### PR TITLE
user-online-status

### PR DESCRIPTION
**Pull request by monikasudol**

### CODE REVIEW

File: src/modules/main/navigator/channels/channel/index.js [hunk 1]
```diff
@@ -20,15 +20,14 @@ export class Channel extends PureComponent {
       </div>
     );
   }
-}
+};
 
-const mapStateToProps = (state) => ({
+const mapStateToProps = () => ({
```
Comment (yuriyyakym) [inline src/modules/main/navigator/channels/channel/index.js]: redundant mapStateToProps

File: src/modules/main/navigator/channels/index.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import FaPlus from 'react-icons/lib/fa/plus-circle';
 import { connect } from 'react-redux';
 import { addNewChannel } from '../../../../state/channels';
 import Channel from './channel';
+import { fetchActiveUsers } from '.././../../../state/active-users';
```
Comment (yuriyyakym) [inline src/modules/main/navigator/channels/index.js]: I guess it shouldn't be here

File: src/state/messages/index.js [hunk 3]
```diff
@@ -6,26 +6,20 @@ const initialState = {
...
     ...state,
-    messages: [{}]
-  }),
-
-  [FETCH_MESSAGES]: (state, { payload }) => ({
-    ...state,
-    loading: true,
-    id: payload.id
+    messages: [],
+    loading: true
```
Comment (yuriyyakym) [inline src/state/messages/index.js]: We do not need this prop

File: src/state/users/selectors.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import { createSelector } from 'reselect';
+
+export const selectUsers = (state) => {
```
Comment (yuriyyakym) [inline src/state/users/selectors.js]: this selector returns nothing

**Review by yuriyyakym — CHANGES_REQUESTED** [review]

File: src/state/users/selectors.js (hunk 4 above)
Comment (monikasudol) [reply src/state/users/selectors.js]: Yes, actually we don't use this

File: src/state/messages/index.js (hunk 3 above)
Comment (monikasudol) [reply src/state/messages/index.js]: Yes, we don't

**Review by yuriyyakym — APPROVED** [review]
Very nice pull request :) 👍

**Pull request merged**